src/components/filter.svelte

```diff
@@ -18,17 +89,39 @@
 		/>
 	</button>
 	<article id="filter-options" class:filter-options-close={!open} class:filter-options-open={open}>
-		<button>Spelsoort</button>
-		<button>Groepen</button>
-		<button>Leerlingenaantal</button>
-		<button>Materialen</button>
+		<FilterButton filterTitle="Spelsoort" on:click={() => (spelsoorten = !spelsoorten)} />
```

Comment:
Hier heb ik het anders gedaan. Ik gebruik [slots](https://github.com/iSirThijs/jungle-gym/blob/development/src/components/Button/ButtonLight.svelte). Maakt niet uit wat we doen. Maar moet wel even consistent zijn. Wat zullen we doen? Meegeven via props of via `<FilterButton>Spelsoort</FilterButton>`. Weet niet wat de best practice is omtrent slots. Kan ook dat dit meer voor pagina templates is. @iSirThijs misschien?